### PR TITLE
2x2x2 ll svg

### PR DESCRIPTION
**Pull request by ticklemepierce**
Dunno how you usually create these but I used https://boxy-svg.com/ in combination with tweaking the 3x3x3 LL code 🤷🏻 if this isn't needed or there is a better way, I would love documentation to assist with it

### CODE REVIEW

**Comment by lgarron** [comment]
Thanks, this is a great PR!

Screenshot 2023-01-02 at 18 23 16

What would you think of moving in the middle square just a bit more, so that all the stickers have more similar sizes?

**Comment by ticklemepierce** [comment]
> What would you think of moving in the middle square just a bit more, so that all the stickers have more similar sizes?

You mean like make the LL squares smaller and the side stickers wider?

**Comment by lgarron** [comment]
> You mean like make the LL squares smaller and the side stickers wider?

Yeah, thanks for the change! I'm thinking equal area for each sticker, though:

Screenshot 2023-01-03 at 18 26 07

File: src/cubing/puzzles/implementations/dynamic/side-events/2x2x2-ll.kpuzzle.svg.ts [hunk 1]
```diff
@@ -9,18 +9,18 @@ export const cube2x2x2LLSVG = `<?xml version="1.0" encoding="UTF-8"?>
...
-    <rect    id="CORNERS-l2-o0" stroke="#000000" style="fill: white" x="32" y="32" width="96" height="96"></rect>
-    <polygon id="CORNERS-l2-o1" stroke="#000000" style="fill: orange" transform="translate(18, 50) scale(-1, -1) translate(-18, -50) " points="4 -28 32 -28 32 96 4 68"></polygon>
-    <polygon id="CORNERS-l2-o2" stroke="#000000" style="fill: #26f" transform="translate(50, 18) scale(1, -1) rotate(90) translate(-50, -18) " points="36 -60 64 -60 64 64 36 36"></polygon>
-    <rect    id="CORNERS-l3-o0" stroke="#000000" style="fill: white" x="32" y="128" width="96" height="96"></rect>
-    <polygon id="CORNERS-l3-o1" stroke="#000000" style="fill: limegreen" transform="translate(50, 238) rotate(90) translate(-50, -238) " points="36 160 64 160 64 284 36 256"></polygon>
-    <polygon id="CORNERS-l3-o2" stroke="#000000" style="fill: orange" transform="translate(18, 206) scale(-1, 1) translate(-18, -206) " points="4 128 32 128 32 252 4 224"></polygon>
+    <rect    id="CORNERS-l0-o0" stroke="#000000" style="fill: white" x="128" y="128" width="88" height="88"></rect>
```
Comment (lgarron) [inline src/cubing/puzzles/implementations/dynamic/side-events/2x2x2-ll.kpuzzle.svg.ts:12]: btw, I can fix this up later, but the `stroke` should go on the surrounding `<g>` to save bytes.

Comment (ticklemepierce) [reply src/cubing/puzzles/implementations/dynamic/side-events/2x2x2-ll.kpuzzle.svg.ts]: it may have gotten messed up on that site I was using

**Comment by lgarron** [comment]
Lookin' snazzy, thanks for the contribution!

```ts

import { TwistyPlayer } from "../../../../cubing/twisty";

document.body.appendChild(
  new TwistyPlayer({
    alg: "R' U2 R U R' U2 L U' R U L'",
    puzzle: "2x2x2",
    visualization: "experimental-2D-LL",
    experimentalStickering: "PLL",
  }),
);

```
Screenshot 2023-01-03 at 20 32 47

**Pull request merged**